sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/src/Generated/Models/PublicNetworkAccess.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.MachineLearning.Models
+{
+    /// <summary> Whether requests from Public Network are allowed. </summary>
+    public readonly partial struct PublicNetworkAccess : IEquatable<PublicNetworkAccess>
```

Comment:
This enum is actually the same as https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/src/Generated/Models/MachineLearningPublicNetworkAccessType.cs

And in referred places like `MachineLearningWorkspacePatch`, the type of `PublicNetworkAccessType` property is changed from `MachineLearningPublicNetworkAccessType` to `PublicNetworkAccess`.

Is that by accident?

Reply:
MachineLearningPublicNetworkAccessType is part of OnlineEndpoint defined in https://github.com/Azure/azure-rest-api-specs/blob/main/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2024-04-01/mfe.json

PublicNetworkAccess is part of workspace patch "WorkspacePropertiesUpdateParameters" defined in https://github.com/Azure/azure-rest-api-specs/blob/main/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2024-04-01/machineLearningServices.json